--- ./src/Vocal.vala
+++ ./src/Vocal.vala
@@ -43,7 +43,7 @@
             build_version_info = Constants.VERSION_INFO;
 
             app_years = "2015-2017";
-            app_icon = "com.github.needle-and-thread.vocal";
+            app_icon = "com.github.needleandthread.vocal";
             app_launcher = "com.github.needle-and-thread.vocal.desktop";
             application_id = "com.github.needle-and-thread.vocal";
 

