--- ./data/com.github.needle-and-thread.vocal.appdata.xml.in
+++ ./data/com.github.needle-and-thread.vocal.appdata.xml.in
@@ -6,7 +6,7 @@
   <project_license>GPL-3.0+</project_license>
   <name>Vocal</name>
   <_summary>A Great Podcast Client for the Modern Free Desktop</_summary>
-  <developer_name>Needle & Thread</developer_name>
+  <developer_name>Needle &amp; Thread</developer_name>
   
   <screenshots>
     <screenshot type="default">
